apps/foundation-demo/src/app/checkout/components/payment/payment-info-form/factories/payment-info-form.factory.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { TestBed } from '@angular/core/testing';
+import { FormGroup, ReactiveFormsModule } from '@angular/forms';
+
+import { PaymentInfoFormFactory } from './payment-info-form.factory';
+
+describe('Foundation Demo | Checkout | Forms | Payment Info Form | Factories | PaymentInfoFormFactory', () => {
+  
+  let paymentInfoFormFactory;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        ReactiveFormsModule
+      ],
+      providers: [PaymentInfoFormFactory]
+    });
+
+    paymentInfoFormFactory = TestBed.get(PaymentInfoFormFactory);
+  });
+
+  it('should be created', () => {
+    expect(paymentInfoFormFactory).toBeTruthy();
+  });
+
+  describe('create', () => {
+    
+    let result: FormGroup;
+    let expectedResult;
+    let paymentInfo;
+
+    describe('when paymentInfo is null', () => {
+      
+      beforeEach(() => {
+        paymentInfo = null;
+        expectedResult = {
```

Comment:
Should use the paymentInfo factory out of `@daffodil/core/testing`

Reply:
done